src/model/cloud-runner/services/cloud-runner-folders.ts

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+import { CloudRunner } from '../..';
+
+export class CloudRunnerFolders {
+  public static readonly repositoryFolder = 'repo';
+
+  // only the following paths that do not start a path.join with another "Full" suffixed property need to start with an absolute /
+
+  public static get uniqueCloudRunnerJobFolderFull(): string {
+    return path.join(`/`, CloudRunnerFolders.buildVolumeFolder, CloudRunner.buildParameters.buildGuid);
+  }
+
+  public static get cacheFolderFull(): string {
+    return path.join(
+      '/',
+      CloudRunnerFolders.buildVolumeFolder,
+      CloudRunnerFolders.cacheFolder,
+      CloudRunner.buildParameters.cacheKey,
+    );
+  }
+
+  public static get builderPathFull(): string {
+    return path.join(CloudRunnerFolders.uniqueCloudRunnerJobFolderFull, `builder`);
+  }
+
+  public static get repoPathFull(): string {
+    return path.join(CloudRunnerFolders.uniqueCloudRunnerJobFolderFull, CloudRunnerFolders.repositoryFolder);
+  }
+
+  public static get projectPathFull(): string {
+    return path.join(CloudRunnerFolders.repoPathFull, CloudRunner.buildParameters.projectPath);
+  }
+
+  public static get libraryFolderFull(): string {
+    return path.join(CloudRunnerFolders.projectPathFull, `Library`);
+  }
+
+  public static get projectBuildFolderFull(): string {
+    return path.join(CloudRunnerFolders.repoPathFull, CloudRunner.buildParameters.buildPath);
+  }
+
+  public static get lfsDirectoryFull(): string {
+    return path.join(CloudRunnerFolders.repoPathFull, `.git`, `lfs`);
+  }
```

Comment:
The way I see it a path is either relative or absolute, but always full.

Do we have any non-full examples? Would they be just the folderName?

Reply:
Agreed, it should be absolute or relative to be clear where needed. 

Uses of full will be removed.